### DISCUSSION TITLE
Unable to rearrange product position between different pages of categories

### DISCUSSION THREAD

**Discussion by darkf3d3** [PrestaShop/PrestaShop#35262] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The feature that allow to rearrange product position into categories it's not fully working.
To rearrange product position you have to drag and drop them.
The problem is that this feature works only on the actual page, so if the category has more than 300 products (the maximum products that can be showed on a single page) you can't move a product to another page.

### Expected behavior

Rearrange product position feature should allow the user to choose any position, even if on a different page

### Steps to reproduce

Back office
Catalog >> Products
Filter products by a category
Then into the Position column click "rearrange"
If category have pagination, there's no way to move a product to another page

### PrestaShop version(s) where the bug happened

8.1

### PHP version(s) where the bug happened

8.1

**Comment by paulnoelcholot** [comment]
Hello @darkf3d3,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you

**Comment by RosaBenouamer** [comment]
Hello @darkf3d3 ,
Where is the position on the product page please ?

**Reply by kpodemski** [reply]
Catalog -> Products, then you need to filter by category and sort by position (if you don't use the newest version). It will show you an interface to sort products.

**Comment by kpodemski** [comment]
Hi @darkf3d3 

It's a known issue, a frustrating one, I agree. There are some modules helping with this by introducing a feature where you can select a bunch of products and select "Move to the first position" or "Move to position X". What do you think, would that work?

**Reply by darkf3d3** [reply]
Hi @kpodemski

Thanks for the advice, I will definitely consider using a module, because the way it is implemented now is unusable for me.
However, I think PS team should consider to fix this or remove completely this feature, because if it doens't allow to sort all products it's useless.

**Reply by kpodemski** [reply]
Yes, I agree, and I understand. I've met a few merchants with the same issue.

cc @MatShir

**Comment by darkf3d3** [comment]
Just for information, today I bought the module on addons and I saw that has been downloaded by 6300 people, so there are several people interested into this feature.
